app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,190 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/facebook_description"
+        app:layout_constraintBottom_toBottomOf="@id/links"
+        app:layout_constraintTop_toTopOf="@id/links"
+        app:srcCompat="@drawable/facebook" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/imageButtonLinkedIn"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/linkedin_description"
+        app:layout_constraintBottom_toBottomOf="@id/links"
+        app:layout_constraintTop_toTopOf="@id/links"
+        app:srcCompat="@drawable/linkedin" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/imageButtonInstagram"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/instagram_description"
+        app:layout_constraintBottom_toBottomOf="@id/links"
+        app:layout_constraintTop_toTopOf="@id/links"
+        app:srcCompat="@drawable/instagram" />
+
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/icon_profile"
+        android:layout_width="113dp"
+        android:layout_height="0dp"
+        android:src="@drawable/icon1"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"
+        app:layout_constraintDimensionRatio="1:1"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.35" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/first_name_sure_name"
+        style="@style/TextAppearance.AppCompat.White18"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/first_name_sure_name"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.65" />
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/career"
+        style="@style/TextAppearance.AppCompat.LogOut14"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/career"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.75" />
+
+    <androidx.appcompat.widget.AppCompatTextView
```

Comment:
On the design, this text has top margin of 16dp to the above group. So this margin either has to be 16dp or dynamically changes according screen dimensions. If this view is currently aligned correctly, this step can be skipped.

Reply:
done